src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -85,9 +85,9 @@
     <!-- gather the test archives for upload -->
     <ItemGroup>
       <ForUpload Include="$(TestArchivesRoot)**/*.zip" />
-      <ForUpload Include="$(AnyOSTestArchivesRoot)*.zip" />
```

Comment:
Running perf with these changes requires furthur changes to https://github.com/karajas/buildtools/blob/99e2773de109d33c5a7a584ea33f2c9b5586390e/src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets#L235, to support the same folder structure. Will update shortly